Fixed incorrect calculation

tomtronic opened the pull request.

tomtronic committed:
Fixed incorrect calculation

markjbrown reviewed (changes requested):
Revert to the original RU cost but change the days to 30 and I'll approve the change.

Storage fix is correct.

Thanks.

tomtronic committed:
Fixed incorrect calculation #2

As suggested, I have updated the calculation with the assumption of 730 hours in a month. Additionally, in the example, I have changed the storage from 1 TB to 0.5 TB, since we require a minimum of 10 RU/s per GB and it is therefore not possible to provision only 10,000 RU/s for 1 TB (or 1,024 GB) of storage - this would require 10,240 RU/s, rounded up 10,300 RU/s. I hope it is okay like this. Please let me know if further changes are needed to merge the PR.
